### PR TITLE
Fix envconsul's tar related issues

### LINKED ISSUE #342: puppet-envconsul doesn't look for tar in all the possible places via PATH
Specifically, it doesnt' look in /bin where it happens to be on Amazon Linux

See reppard/puppet-envconsul#2

### PR DESCRIPTION
**Pull request by gozer**
Fixes #342

### CODE REVIEW

**Comment by limed** [comment]
r+

File: nubis/puppet/consul.pp [hunk 1]
```diff
@@ -36,11 +36,9 @@
     group            => 'root',
 }
 
-package { "tar":
```
Comment (limed) [inline nubis/puppet/consul.pp]: Removing this line will break the require in line 41

Comment (gozer) [inline nubis/puppet/consul.pp]: Actually, no, it fixes the problem!

The issue is that some other puppet module, consul_template, IIRC, already includes package { "tar", and this
was causing a package redefined error.

So, this is the correct fix, actually, even if I recognize it's not obvious.

Comment (limed) [inline nubis/puppet/consul.pp]: That kind of sucks that its not obvious, I would at least write down a note or a comment up top to say where `package { 'tar': }` is coming from

**Comment by limed** [comment]
r+

**Pull request merged**